File: server/src/Mentora.Application/DTOs/WorkspaceResponse.cs
namespace Mentora.Application.DTOs;

public record WorkspaceResponse
{
    public Guid Id { get; set; }
    public string Name { get; set; } = string.Empty;
    public string Logo { get; set; } = string.Empty;
    public string PrimaryColor { get; set; } = string.Empty;
    public string SecondaryColor { get; set; } = string.Empty;
    public string BigBanner { get; set; } = string.Empty;
    public string SmallBanner { get; set; } = string.Empty;
    public bool Active { get; set; }
    public string Url { get; set; } = string.Empty;
}

File: mentora-web/src/app/services/responses/workspace.response.ts
export interface Workspace {
  id: string;
  name: string;
  logo: string;
  primaryColor: string;
  secondaryColor: string;
  bigBanner: string;
  smallBanner: string;
  active: boolean;
  url: string;
  createdAt: string;
  updatedAt?: string;
}
